modules/ROOT/pages/clustering/databases.adoc

```diff
@@ -225,6 +225,123 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
 +----------------------------------------------------------------------------------------------------------------------------------------+
 ----
 
+[role=label--new-5.24]
+[[recreate-databases]]
+== Recreate a database
+
+Neo4j 5.24 introduces the xref:reference/procedures.adoc#procedure_dbms_cluster_recreateDatabase[`dbms.cluster.recreateDatabase()`] procedure, which allows you:
+
+* To change the database store to a specified backup, while keeping all the associated privileges for the database.
+
+* To make your database write-available again after it has been lost (for example, due to a disaster).
+// See xref:clustering/disaster-recovery.adoc[] for more information.
+ 
+[CAUTION]
+====
+The recreate procedure works only for real user databases and not for composite databases, or the `system` database.
+
+Remember that the recreate procedure results in downtime while the stores get updated.
+The time is unbounded and may depend on different factors -- for example, the size of the store, network speed, etc.
+====
+
+The database in question can be in an `online` or `offline` state when it is recreated, but a successful operation starts the database regardless of its previous state.
+
+If CDC is enabled for the database, it will still be enabled after recreating the database, however the chain will be broken.
+See link:https://neo4j.com/docs/cdc/current/backup-restore/[CDC documentation -> Restore backups and snapshots] for more details.
```

Comment:
Okay, so I think we should not link to the docs I suggested last time. That handles what the user can see and should do if the database was restored. In the case of recreate, we change database id , so effectively it is a new database. Therefore we should link to how to set CDC up from the beginning. My suggestion is this: 

If CDC is enabled for the database, the chain will stop when the database is recreated. While the recreated database will still have CDC enabled, follow the link:https://development.neo4j.dev/docs/cdc/current/existing-databases/[Initialize CDC applications from an existing database] guide to get it working again.